beetsplug/acousticbrainz.py

```diff
@@ -141,9 +140,17 @@ def __init__(self):
         self.config.add({
             'auto': True,
             'force': False,
-            'tags': []
+            'tags': [],
+            'hostname': ''
         })
 
+        self.hostname = self.config['hostname'].as_str()
+        if self.hostname:
+            if self.hostname[-1] != '/':
```

Comment:
I'm considering a couple things about this:
- I think this option should be renamed to `base_url` since it needs to include the scheme and possibly port.
- Should more validation happen here, like verifying the option _does_ include a scheme? Only `http` or `https` would be supported.